### PR TITLE
worked on update user profile page and jsp paths

### PR DESCRIPTION
**Pull request by mksyms**

### CODE REVIEW

**Comment by MontealegreLuis** [comment]
Please fix the merge conflicts. Ping me again, once it's done, so I can start reviewing.

Thank you!

File: src/main/java/com/codeup/adlister/controllers/LoginServlet.java [hunk 1]
```diff
@@ -18,19 +18,17 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
...
     }
 
     protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
         String username = request.getParameter("username");
         String password = request.getParameter("password");
         User user = DaoFactory.getUsersDao().findByUsername(username);
-        System.out.println("test");
+
         if (user == null || !BCrypt.checkpw(password, user.getPassword())) {
-            System.out.println("inside if statment");
```
Comment (MontealegreLuis) [inline src/main/java/com/codeup/adlister/controllers/LoginServlet.java:30]: Thank you for removing the `sout`s

**Review by MontealegreLuis — APPROVED** [review]
It looks good, thank you!

File: src/main/java/com/codeup/adlister/controllers/LoginServlet.java (hunk 1 above)
Comment (mksyms) [reply src/main/java/com/codeup/adlister/controllers/LoginServlet.java:30]: I'm still not entirely sure on merges 😬 Would it be possible to have a review or example of how we should collaborating before Capstone? Thank you so much for looking over!

Comment (MontealegreLuis) [reply src/main/java/com/codeup/adlister/controllers/LoginServlet.java:30]: Sure, no problem! 😄

**Pull request merged**